.github/workflows/persistent-devnet.yml

```diff
@@ -0,0 +1,108 @@
+name: Deploy Persistent Devnet
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "Release tag, branch name or any revision."
+        required: true
+        type: string
+
+env:
+  NIX_NIXPKGS_CHANNEL: https://nixos.org/channels/nixpkgs-22.05-darwin
+  CACHIX_COMPOSABLE: composable-community
+  CACHIX_COMPRESSION_LEVEL: 3
+  NIXOPS_STATE_URL: gs://composable-state
+  NIXOPS_STATE_FILE: persistent.deployment.nixops
+
+jobs:
+  cache-devnet:
+    name: "Cache Persistent Devnet"
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    concurrency:
+       group: cache-devnet-persistent
```

Comment:
group may conflict with global group until prefixed by workflow (like in build editor)

Reply:
so you cannot run you own nixops from branch in parallel

Reply:
It's done on purpose because the state file is shared and this tag must then be unique if the state file is used by another workflow